scripts/module_list/module_overview.py

```diff
@@ -0,0 +1,175 @@
+import os
+import subprocess
+from typing import Union, Tuple
+from mdutils.mdutils import MdUtils
+import numpy as np
+
+
+# --------------------------------------------------------------------------------------------------------
+# Module bash API
+# --------------------------------------------------------------------------------------------------------
+
+def module(*args, filter_fn=lambda x: x) -> np.ndarray:
+    """
+    API to call the module command.
+
+    @param args: Extra arguments for the module command.
+    @param filter_fn: Filter function on the ouput.
+    @return:
+    """
+    lmod = os.getenv('LMOD_CMD')
+    proc = subprocess.run(
+        [lmod, "python", "-t"] + list(args),
```

Comment:
use `--terse` instead of `-t`, to make this "self-documenting", `-t` could be anything